.scalafmt.conf

```diff
@@ -0,0 +1,10 @@
+style = default
+align.openParenCallSite=false
+align.openParenDefnSite=false
+align.tokens = []
```

Comment:
align=none should do this for you

Reply:
This could be changed in scalameta too then.